cli/diff.rs

```diff
@@ -0,0 +1,151 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+use crate::colors;
+use dissimilar::{diff, Chunk};
+use std::path::PathBuf;
+
+macro_rules! print_add {
+  () => {
+    print!("{}", colors::green_bold("+".to_string()));
+  };
+}
+
+macro_rules! fmt_add_text {
+  ($x:expr) => {
+    format!("{}", colors::green($x));
+  };
+}
+
+macro_rules! print_rem {
+  () => {
+    print!("{}", colors::red_bold("-".to_string()));
+  };
+}
+
+macro_rules! fmt_rem_text {
+  ($x:expr) => {
+    format!("{}", colors::red($x));
+  };
+}
+
+macro_rules! fmt_add_text_highlight {
+  ($x:expr) => {
+    format!("{}", colors::white_on_green($x));
+  };
+}
+
+macro_rules! fmt_rem_text_highlight {
+  ($x:expr) => {
+    format!("{}", colors::white_on_red($x));
+  };
+}
```

Comment:
Generally we try to avoid macros unless they're really necessary. I think in this case you might be able to remove them and replace with functions.

Reply:
I replaced them with equivalent functions